ArmTemplates/keyvault-secrets.md

```diff
@@ -0,0 +1,87 @@
+# KeyVault secrets
+
+Creates secrets within the key vault
+
+## Paramaters
+
+keyVaultName: (required) string
+
+Name of key vault. Will be created in the same resource group as the script is run and in the default location for resource group.

```

Comment:
KeyVault must already exist or be defined separately in a template linked to or nested within the same parent.  Secrets will be created in this KeyVault.

Reply:
Done